sdk/CHANGELOG.md

```diff
@@ -19,6 +19,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security -->
 
+## 0.5.0 - 2023-MM-DD
+
+### Changed
+
+- `SecretManage::sign_evm` renamed to `sign_secp256k1_ecdsa`;
```

Comment:
Really sorry but I was actually unable to release because of stronghold, just rename the existed added entry

Reply:
Done